src/test/java/com/mercadopago/client/point/PointClientIT.java

```diff
@@ -0,0 +1,92 @@
+package com.mercadopago.client.point;
+
+import static com.mercadopago.net.HttpStatus.CREATED;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.mercadopago.BaseClientIT;
+import com.mercadopago.exceptions.MPApiException;
+import com.mercadopago.exceptions.MPException;
+import com.mercadopago.resources.point.PointPaymentIntent;
+import java.math.BigDecimal;
+import org.junit.jupiter.api.Test;
+
+/** PointClientIT class. */
+class PointClientIT extends BaseClientIT {
+  private final String deviceId = "GERTEC_MP35P__8701012051267123";
+
+  private final PointClient client = new PointClient();
+
+  @Test
+  void createPaymentIntentSuccess() {
+    try {
+      PointPaymentIntentRequest request = buildPaymentIntentRequest();
+
+      PointPaymentIntent paymentIntent = client.createPaymentIntent(deviceId, request);
+
+      assertNotNull(paymentIntent.getResponse());
+      assertEquals(CREATED, paymentIntent.getResponse().getStatusCode());
+      assertPaymentIntentFields(paymentIntent);
+    } catch (MPApiException mpApiException) {
+      fail(mpApiException.getApiResponse().getContent());
+    } catch (MPException mpException) {
+      fail(mpException.getMessage());
+    }
```

Comment:
Interessante esse recurso, mas ele não deveria ser utilizado em um teste de falha? Por que nesse caso não cairá no catch... e não testará nada.

Reply:
Nossa estratégia com os testes do SDK é ter testes utilizando mocks para as respostas das APIs (arquivos com final `*Test.java` e testes integrados, que fazem chamadas reais para as APIs (arquivos com final `*IT.java`).

Os testes de integração cobre sim todos os cenários, porém eles podem falhar por diversos motivos, seja por instabilidade de alguma API, por access token de outro país, entre outros, por isso escrevemos esses testes para comprovar que o endpoint está funcionando e para futuros testes, mas não rodamos ele na CI, o que o torna um teste não bloqueante. 
Então sim, existe a possiblidade de um teste IT mesmo de cenário feliz falhar, por isso utilizamos o `fail()`, para indicar essa falha seja lá qual for o motivo e nos dar a possiblidade de re-testar e olhar para tal endpoint com um pouco mais de cuidado.

Já sobre os testes que utilizam mocks, esses mocks foram construídos com o máximo de fidelidade possível de uma resposta da API em questão, capturados após teste em chamada direta à API, o que nos da uma segurança maior sobre o funcionamento do endpoint do SDK.

Além disso, temos testes para as classes que montam as requisições e para a classe Serializer, o que nos passa ainda mais segurança sobre a eficiência do SDK. 

Dito isso, creio que agora não caiba incluir esse novo tipo de fluxo (fluxos tristes), mas talvez valha a pena discutir se essa é uma necessidade para incluirmos em todos os clients no futuro. O que acha?